overcast_to_sqlite/datastore.py

```
@@ -194,6 +195,15 @@ def save_feed_and_episodes(
         episodes: list[dict],
     ) -> None:
         """Upsert feed and episodes into database."""
+        limit_days = os.getenv("OVERCAST_LIMIT_DAYS")
+        if limit_days:
+            cutoff_date = datetime.datetime.now(tz=datetime.UTC) - datetime.timedelta(days=int(limit_days))
```

Ruff (0.11.9)

Line too long (107 > 88)

(E501)